libyul/backends/wasm/WasmDialect.cpp

```diff
@@ -56,10 +56,17 @@ WasmDialect::WasmDialect():
 	addFunction("i64.eqz", 1, 1);
 	m_functions["i64.eqz"_yulstring].returns.front() = "i32"_yulstring;
 
+	addFunction("i64.clz", 1, 1);
+	m_functions["i64.clz"_yulstring].returns.front() = "i64"_yulstring;
```

Comment:
Need to implement this opcode in the interpreter.

Reply:
done